lib/commander/user_interaction.rb

```diff
@@ -24,6 +24,44 @@ module UI
       include Growl
     end
 
+    #--
+    # Mimic growl in linux with libnotify
+    #++
+
+    begin
+      require 'libnotify'
+    rescue LoadError
+      # Do nothing
+    else
+      Libnotify.icon_dirs << '/usr/share/icons/gnome/*/'
```

Comment:
Is this directory available on all desktop linux variants?

Reply:
That is the path on debian/ubuntu/fedora, provided gnome is installed. An additional check could be put in for the existence of that folder, and omit the icons if it is not present. I'm not aware of a more generic or reliably installed icon set, unless you wanted to include a few image assets in this gem.

Reply:
I guess this is probably fine... what is the failure mode if that directory is not present or if the requested icon image can't be found? Does the notice still display, just without an icon? Or is there an error of some kind?